hermes-common/src/main/java/pl/allegro/tech/hermes/common/di/factories/PrometheusMeterRegistryFactory.java

```diff
@@ -55,4 +61,11 @@ private void registerZookeeperReporter(PrometheusMeterRegistry meterRegistry) {
         new ZookeeperCounterReporter(meterRegistry, counterStorage, prefix)
                 .start(parameters.zookeeperReportPeriod().toSeconds(), TimeUnit.SECONDS);
     }
+
+    private void registerJvmMetrics(MeterRegistry meterRegistry) {
+        new JvmMemoryMetrics().bindTo(meterRegistry);
+        new JvmGcMetrics().bindTo(meterRegistry);
+        new JvmThreadMetrics().bindTo(meterRegistry);
+        new FileDescriptorMetrics().bindTo(meterRegistry);
```

Comment:
I believe there's no need to report metrics related to file descriptors, as they are typically exposed by the environment in which the application runs (e.g. `container_file_descriptors` in k8s).

Reply:
done